ui/static/ui/css/mit-lore.css

```diff
@@ -6,6 +6,7 @@
 
 *::after, *::before {
     content: '';
```

Comment:
Changes like these should be made specific to avoid possibly breaking other parts of the page like #530